server/src/termdb.gdc.status.js

```diff
@@ -0,0 +1,45 @@
+import path from 'path'
+import ky from 'ky'
+import serverconfig from '#src/serverconfig.js'
+
+/**
+ * Tests the GDC API status and returns true if the status is OK.
+ * If serverconfig.features.runRemainingWithoutAwait is false, it will throw an error if the status is not OK.
+ * @param ds
+ * @returns {Promise<boolean>}
+ */
+export async function testGDCApiStatus(ds) {
+	try {
+		const status = await getApiStatus(ds, 4)
```

Comment:
The number of retries should be much more. Also, using the default 1 second retryDelay is too short, should be at least 10 minutes. The GDC API under maintenance status is very unlikely to change every second.

Reply:
Should the server wait for GDC API to come back online before it starts, or should it restart and keep checking for GDC API to come back online?

Reply:
The latter, should restart and keep checking. The server should try to load the GDC status the first time, if it fails (not `ok`) and the `runRemainingAwait` is true, then keep retrying without blocking the loading of other datasets.

Reply:
For comparison, there are a few active loops to check GDC BAM cache and also separately, data release version to refresh the cache as needed. Both of these run without requiring the server to restart in case of needing a refresh, so I'm thinking along similar lines with regards to the status check, to have an active loop to try to automatically recover the GDC dataset loading without requiring a full server restart.